tools/image_processing/yolo/json2yolosegment.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="json2yolosegment" name="Convert AnyLabeling JSON to YOLO text" version="@VERSION@+galaxy0" profile="24.2">
+    <description>with ultralytics</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator" />
+    <command detect_errors="aggressive">
+    <![CDATA[
+      mkdir ./input ./output &&
+		    
+      #for $i, $filename in enumerate($in_json):
+          ln -s '$filename' './input/${filename.element_identifier}' &&
```

Comment:
you are not using `$i` here, isn't it? You might want to use i to make sure the names are unique?